docs/integrations/digitalocean/gslb.mdx

```diff
@@ -0,0 +1,315 @@
+
+```bash
+curl \
+  -X POST https://api.ngrok.com/backends/tunnel_group \
+  -H "Authorization: Bearer ${NGROK_API_KEY}" \
+  -H "Content-Type: application/json" \
+  -H "Ngrok-Version: 2" \
+  -d '{"labels":{"dc":"<LOCATION_NAME>"}}'
+```
+
+Export the `id`, beginning with `bkdtg_...`, returned from the ngrok API for each Tunnel Group you create.
+
+```bash
+export BACKEND_ID_01=bkdtg_...
+export BACKEND_ID_02=bkdtg_...
+export BACKEND_ID_03=bkdtg_...
+```
+
+## **Step 4**: Create a Weighted Backend and Route on your Edge {#create-weighted-backend}
+
+With a [Weighted Backend](/docs/network-edge/edges.mdx#weighted), you can specify custom weights for Tunnel Group backends, shaping precisely how to load-balance traffic across the infrastructure you've distributed around the globe. Without a Weighted Backend, the ngrok Edge and [GSLB](https://ngrok.com/docs/network-edge/gslb/) will always favor the Point of Presence (PoP) and VM nearest the user making requests.
+
+Create a Weighted Backend at the [`/backends/weighted` endpoint](https://ngrok.com/docs/api/resources/weighted-backends/) using your Tunnel Group backend `id`s exported from the previous step. The `curl` command below creates equal weighting for your backends, but you can use integers between `0-10000` to configure precise proportional weights.
+
+```bash
+curl \
+	-X POST https://api.ngrok.com/backends/weighted \
+	-H "Authorization: Bearer ${NGROK_API_KEY}" \
+	-H "Content-Type: application/json" \
+	-H "Ngrok-Version: 2" \
+	-d '{"backends":{"'${BACKEND_ID_01}'":1,"'${BACKEND_ID_02}'":1,"'${BACKEND_ID_03}'":1}}'
+```
+
+Export the `id` of your new Weighted Backend.
+
+```bash
+export WEIGHTED_BACKEND_ID=bkdwd_...
+```
+
+You can now create an catch-all Edge Route at `/` using your Weighted Backend using the [`/edges/https/{EDGE_ID}/routes` endpoint](https://ngrok.com/docs/api/resources/edges-https-routes/).
+
+```bash
+curl \
+  -X POST "https://api.ngrok.com/edges/https/${EDGE_ID}/routes" \
+  -H "Authorization: Bearer ${NGROK_API_KEY}" \
+  -H "Content-Type: application/json" \
+  -H "Ngrok-Version: 2" \
+  -d '{"match":"/","match_type":"path_prefix","backend":{"enabled":true,"backend_id":"'${WEIGHTED_BACKEND_ID}'"}}'
+```
+
+If you'd like to do optional step #7, you will need to export a variable for the route you just created:.
+
+```bash
+export ROUTE_ID=edghtsrt_...
+```
+
+:::tip Make sure you grab the right id
+The command starts with two ID values (`edge_id` and `id`), be sure to use `id` with the value that starts with `edghtsrt`.
+:::
+
+You now have an active domain, Edge, Tunnel Group backends, a Weighted Backend, and a route. If you open a new tab and navigate to `https://<YOUR_NGROK_DOMAIN>` right now, you'll see an error message from ngrok. You've successfully reserved a domain and created an edge, but don't yet have ngrok agents or active tunnels.
+
+![An example of an ngrok domain and Edge with no matching tunnels](img/edge-no-tunnels.png)
```

Comment:
It's a bit of a nitpick, but it's a little confusing that the screenshot has **Linode** in it instead of **DigitalOcean**. A quick Photoshop might be enough of a fix :)

Reply:
Same goes for the other screenshot near the end.

Reply:
updated graphics